celestia/chain.json

```diff
@@ -83,13 +83,15 @@
       },
       {
         "name": "v2",
+        "height": 2371495,
         "recommended_version": "v2.1.2",
         "compatible_versions": [
           "v2.0.0",
           "v2.1.0",
           "v2.1.1",
           "v2.1.2"
-        ]
+        ],
+        "next_version_name": ""
```

Comment:
lgtm as long as this can be empty. it will be v3, but i think we can make that change later. wdyt @rootulp ?

Reply:
@jcstein we tend to set a dummy blank "next_version_name", so if someone goes to update they might catch that it needs filled! this is good to go! Thanks for catching and inquiring though!

Reply:
anytime! thanks for following up 😄